README.md

```diff
@@ -1,13 +1,184 @@
+Request<Balance> activationRequest = account.activate(passphrase)
+activationRequest.run(new ResultCallback<Void>() {
+    @Override
+    public void onResult(Void result) {
+        Log.d("example", "Account is activated");
+    }
+
+    @Override
+    public void onError(Exception e) {
+        e.printStackTrace();
+    }
+});
+``` 
+For a complete example of this process, take a look at Sample App `OnBoarding` class.
+
+### Public Address
+Your account can be identified via it's public address. To retrieve the account public address use:
+```java
+account.getPublicAddress();
+```
+
+
+### Retrieving Balance
+To retrieve the balance of your account in KIN call the `getBalance` method: 
+```java
+Request<Balance> balanceRequest = account.getBalance();
+balanceRequest.run(new ResultCallback<Balance>() {
+
+    @Override
+    public void onResult(Balance result) {
+        Log.d("example", "The balance is: " + result.value(2));
+    }
+
+    @Override
+        public void onError(Exception e) {
+            e.printStackTrace();
+        }
+});
+```
+
+### Transfering KIN to another account
+To transfer KIN to another account, you need the public address of the account you want 
+to transfer the KIN to. 
+
+The following code will transfer 20 KIN to account "#AB12349ACF123". 
+```java
+String toAddress = "#AB12349ACF123";
+String passphrase = "yourPassphrase";
+BigDecimal amountInKin = new BigDecimal("20");
+
+
+transactionRequest = account.sendTransaction(toAddress, getPassphrase(), amount);
+transactionRequest.run(new ResultCallback<TransactionId>() {
+
+    @Override
+        public void onResult(TransactionId result) {
+            Log.d("example", "The transaction id: " + result.toString());
+        }
+
+        @Override
+        public void onError(Exception e) {
+            e.printStackTrace();
+        }
+});
+```
+
+### Sync vs Async
+
+Asynchronous requests are supported by our `Request` object. The `request.run()` method will perform the request on a serial 
+background thread and notify success/failure using `ResultCallback` on the android main thread. 
+In addition, `cancel(boolean)` method can be used to safely cancel requests and detach callbacks.
+
+
+A synchronous version of these methods is also provided. Make sure you call them in a background thread.
+
+```java
+try {
+    account.getBalanceSync();
+}
+catch (OperationFailedException e) {
+   // something went wrong - check the exception message
+}
+
+try {
+    account.sendTransactionSync(toAddress, passphrase, amountInKin);
+}
+catch (PassphraseException e){
+    // the passphrase used was wrong
+}
+catch (OperationFailedException e){
+    // something else went wrong - check the exception message
+} 
+```
+
+### Sample Application 
+For a more detailed example on how to use the library please take a look at our [Sample App](sample/).
+
+## Testing
+
+Both Unit tests and Android tests were provided, Android tests includes integration tests with Stellar test network, 
```

Comment:
Typos.
Both Unit tests and Android tests are provided. 
Android tests include integration tests that run on the Stellar testnet. These tests are marked as @LargeTest`, because they are time consuming.

Reply:
sentence revised